### ISSUE TITLE
版面分析训练10k图片后，会出现漏检和检测框宽度小于实际的情况，使用微软的dit模型训练后能检测出来

### ISSUE THREAD

**Issue by zhuxiaobin** [PaddlePaddle/PaddleOCR#11566]
picodet模型:
宽度小
![1243a3bccb77b95928ebe5c77049445](https://github.com/PaddlePaddle/PaddleOCR/assets/5206194/f5702b3d-d118-4197-980c-0c7aac07d86c)
漏检
![15d9f89a422888fc27b1555f8de157c](https://github.com/PaddlePaddle/PaddleOCR/assets/5206194/10bfaa24-c86d-4874-bb35-d29fefbb4e74)

dit模型:
宽度正常
![3df7b84c962a3a911fef063eb6b2071](https://github.com/PaddlePaddle/PaddleOCR/assets/5206194/8fcd8891-6e05-465e-8b95-a7daecb77a35)
没有漏检
![871fa7434df03cada1d9d3225f8e0e4](https://github.com/PaddlePaddle/PaddleOCR/assets/5206194/9aad6de9-becf-4955-8e27-313ec721da32)

picodet训练的配置文件如下：
_BASE_: [
  '../../configs/runtime.yml',
  '../../configs/picodet/legacy_model/_base_/picodet_esnet.yml',
  '../../configs/picodet/legacy_model/_base_/optimizer_100e.yml',
  '../../configs/picodet/legacy_model/_base_/picodet_640_reader.yml',
]

pretrain_weights: ppstructure/picodet_lcnet_x1_0_fgd_layout_cdla.pdparams
weights: ppstructure/ml/version_id/checkpoint/best_model
find_unused_parameters: True
use_ema: true
cycle_epoch: 10
snapshot_epoch: 1
epoch: 200

LearningRate:
  base_lr: 0.05
  schedulers:
  - name: CosineDecay
    max_epochs: 100
  - name: LinearWarmup
    start_factor: 0.1
    steps: 300

PicoDet:
  backbone: LCNet
  neck: CSPPAN
  head: PicoHead
  nms_cpu: True

LCNet:
  scale: 1.0
  feature_maps: [3, 4, 5]

metric: COCO
num_classes: 14

TrainDataset:
    name: COCODataSet
    image_dir: images 
    anno_path: result.json 
    dataset_dir: ./ppstructure/ml/version_id/train
    data_fields: ['image', 'gt_bbox', 'gt_class', 'is_crowd']

EvalDataset:
    name: COCODataSet
    image_dir: images 
    anno_path: result.json
    dataset_dir: ./ppstructure/ml/version_id/val

TestDataset:
  !ImageFolder
    anno_path: ./ppstructure/ml/version_id/val/result.json

worker_num: 8 
eval_height: &eval_height 800
eval_width: &eval_width 608
eval_size: &eval_size [*eval_height, *eval_width]

TrainReader:
  sample_transforms:
  - Decode: {}
  - RandomCrop: {}
  - RandomFlip: {prob: 0.5}
  - RandomDistort: {}
  batch_transforms:
  - BatchRandomResize: {target_size: [[768, 576], [800, 608], [832, 640]], random_size: True, random_interp: True, keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_size: 24 
  shuffle: true
  drop_last: true
  collate_batch: false

EvalReader:
  sample_transforms:
  - Decode: {}
  - Resize: {interp: 2, target_size: [800, 608], keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_transforms:
  - PadBatch: {pad_to_stride: 32}
  batch_size: 8 
  shuffle: false

TestReader:
  inputs_def:
    image_shape: [1, 3, 800, 608]
  sample_transforms:
  - Decode: {}
  - Resize: {interp: 2, target_size: [800, 608], keep_ratio: False}
  - NormalizeImage: {is_scale: true, mean: [0.485,0.456,0.406], std: [0.229, 0.224,0.225]}
  - Permute: {}
  batch_transforms:
  - PadBatch: {pad_to_stride: 32}
  batch_size: 1
  shuffle: false

版面分析训练10k图片后，会出现漏检和检测框宽度小于实际的情况，使用微软的dit模型训练后能检测出来，宽度也正常。上面是我的训练配置文件，请问是哪里有问题么？

**Comment by cuicheng01** [comment]
picodet本身是一个小模型，如果希望精度更高，可以尝试使用较大的模型

**Comment by zhuxiaobin** [comment]
@cuicheng01 有没有推荐的模型呢？

**Comment by a31413510** [comment]
@zhuxiaobin 建议使用yolo试试呢，版面分析的第二个

**Comment by a31413510** [comment]
@zhuxiaobin 建议使用yolo试试呢，版面分析的第二个
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/docs/models_list.md
